### PR TITLE
Update to NServiceBus.Core v7

### PR DESCRIPTION
**Pull request by timbussmann**
I did not switch to the new csproj based packaging approach as I noticed some custom packaging logic which includes ILMerge. I'll leave this to the maintainers to decide how to deal with this, but I'd suggest to remove ILMerging if you want to make this package available on net core.

Also had to remove one acceptance test, as it tests transport specific behavior related to non-existent queues (which works when using the learning transport).

ping @Particular/metrics-maintainers

### CODE REVIEW

**Comment by mikeminutillo** [comment]
Is the ILMerge still needed? It was used to embed Metrics.NET but I think we've moved away from it.

**Comment by Scooletz** [comment]
We did but not fully @mikeminutillo . The `queuelength` is still reported with Metrics.NET. With this PR https://github.com/Particular/NServiceBus.Metrics/pull/48 it's becoming more and more probable that we'll drop it totally and just leave our pure bits & bytes (TM) on the wire.

File: packaging/nuget/NServiceBus.Metrics.nuspec [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <description>Hooks into the NServiceBus pipeline and computes metrics like Processing Time and reports it.</description>
     <tags>$tags$</tags>
     <dependencies>
-      <dependency id="NServiceBus" version="[6.2.0, 7.0.0)" />
+      <dependency id="NServiceBus" version="[7.0.0, 8.0.0)" />
```
Comment (SimonCropp) [inline packaging/nuget/NServiceBus.Metrics.nuspec]: should this be an unstable lower range?

Comment (timbussmann) [reply packaging/nuget/NServiceBus.Metrics.nuspec]: yeah makes sense during beta phase

**Comment by timbussmann** [comment]
pushed updates referencing the available core beta packages and addressing simons comment.

**Comment by timbussmann** [comment]
fyi: I'm going on vacation, so I'll not update this PR. Please feel free to hijack this PR.

**Comment by SimonCropp** [comment]
having git issues with this branch. so ported to a new clean one https://github.com/Particular/NServiceBus.Metrics/pull/55

**Pull request closed without merging**